src/core/TabulatedPotential.hpp

```diff
@@ -28,18 +28,37 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <cassert>
 #include <vector>
 
+/** Evaluate forces and energies using a custom potential profile.
+ *
+ *  Forces and energies are evaluated by linear interpolation. The curves
+ *  @ref force_tab and @ref energy_tab must be sampled uniformly between
+ *  @ref minval and @ref maxval.
+ */
 struct TabulatedPotential {
+  /** Position on the x-axis of the first tabulated value. */
   double minval = -1.0;
+  /** Position on the x-axis of the last tabulated value. */
   double maxval = -1.0;
+  /** %Distance on the x-axis between tabulated values. */
```

Comment:
`%`

Reply:
The role of this symbol is to prevent the creation of an autolink to the `Distance` struct defined in `/src/core/short_range_loop.hpp` ([Doxygen doc](http://www.doxygen.nl/manual/faq.html#faq_class))